lib/utility/moduleHelper.ts

```diff
@@ -28,14 +43,34 @@ export function checkIfModuleIsInstalled(
 
 let inMemoryCache: TInstalledModules | null = null;
 
-function resolveListOfInstalledRootModules() {
+function resolveListOfInstalledRootModules(): TInstalledModules {
     if (inMemoryCache) {
         return inMemoryCache;
     }
 
-    const stdout = execSync('npm list --json --depth=0');
+    const command = 'npm list --json --depth=0';
+    let commandOutput: string | null;
+
+    try {
+        commandOutput = execSync(command).toString();
+    } catch (error) {
+        warn(
+            `Command '${command}' resulted in an error. Perhaps there are unmet peer dependencies?`
+        );
+
+        console.error(error);
```

Comment:
Zou dit niet via een (centrale) logger o.i.d. moeten lopen?

Reply:
Wat bedoel je met een centrale logger? Bedoel je geabstraheerd in een losse module voor centraal onderhoud oid.?